examples/plugins/pf-ring/plugin.c

```diff
@@ -0,0 +1,35 @@
+#include "suricata-plugin.h"
+
+#include "source-pfring.h"
+#include "runmode-pfring.h"
+#include "util-device.h"
+
+static char *source_name = "pfring-plugin";
+
+void InitSourcePlugin(const char *args, int plugin_slot, int receive_slot, int decode_slot)
+{
+    LiveBuildDeviceList("pfring");
+    RunModeIdsPfringPluginRegister(plugin_slot);
+    TmModuleReceivePfringPluginRegister(receive_slot);
+    TmModuleDecodePfringPluginRegister(decode_slot);
+}
+
+void SCPluginInit(void)
+{
+    SCLogNotice("SCPluginInit");
+    SourcePlugin *plugin = SCCalloc(1, sizeof(SourcePlugin));
```

Comment:
we use 'source' in suri, but I wonder if 'capture' might make more sense to ppl

Reply:
Capture is more clear.  Will move to that.